core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -189,9 +189,9 @@ public void testSchemaValidation() throws Exception {
     PythonEvaluator.Config config = new PythonEvaluator.Config(
       "def transform(x, emitter, context):\n" +
         " y = {}\n" +
-        "  y['intField'] *= 1024\n" +
-        "  y['longField'] *= 1024\n" +
-        "  emitter.emit(y)",
+        "y['intField'] *= 1024\n" +
```

Comment:
missing a leading space. Also, should this not be `y` is a (shallow?) copy of `x`? Those fields are not defined in `y`.